examples/streamReceivingServer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Legionth\React\Http\HttpServer;
+use React\Socket\Server as Socket;
+use RingCentral\Psr7\Response;
+use Psr\Http\Message\RequestInterface;
+use React\Promise\Promise;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$callback = function(RequestInterface $request) {
+
+    $body = $request->getBody();
+
+    return new Promise(function ($resolve, $reject) use ($body) {
+        $bodyContent = '';
+
+        $body->on('data', function ($chunk) use ($resolve, &$bodyContent) {
+            $bodyContent .= $chunk;
+        });
+
+            $body->on('end', function () use (&$bodyContent, $resolve) {
```

Comment:
Indentation is a bit off here. Also, I wonder why this is different from the example in the README?